### PR TITLE
Added a waggledance_logs_retention_days variable to set retention_day…

### PR DESCRIPTION
**Pull request by givanovexpe**
…s for cloudwatch group and set the default to 30

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -22,6 +22,12 @@ variable "waggledance_version" {
   default     = "3.3.2"
 }
 
+variable "waggledance_logs_retention_days" {
+  description = "Log retention in days for the Waggle Dance cloudwatch log group."
```
Comment (massdosage) [inline variables.tf:26]:
```suggestion
  description = "Log retention in days for the Waggle Dance CloudWatch log group."
```

File: README.md [hunk 2]
```diff
@@ -34,7 +34,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | vpc_id | VPC ID. | string | - | yes |
 | wd_ecs_task_count | Number of ECS tasks to create. | string | `1` | no |
 | wd_instance_type | Waggle Dance instance type, possible values: `ecs`,`k8s`. | string | `ecs` | no |
-| waggledance_version | Waggle Dance version to install on EC2 nodes | string | `3.3.2` | no |
+| waggledance_version | Waggle Dance version to install on EC2 nodes. | string | `3.3.2` | no |
+| waggledance_logs_retention_days | Log retention in days for the Waggle Dance cloudwatch log group. | string | `30` | no |
```
Comment (massdosage) [inline README.md:38]:
```suggestion
| waggledance_logs_retention_days | Log retention in days for the Waggle Dance CloudWatch log group. | string | `30` | no |
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.1.1] - TBD
+
+### Added
+- Added waggledance_logs_retention_days variable that sets the retention_days of the waggledance cloudwatch group
```
Comment (massdosage) [inline CHANGELOG.md:9]:
```suggestion
- Added `waggledance_logs_retention_days` variable that sets the `retention_days` of the Waggle Dance CloudWatch group.
```

File: README.md (hunk 2 above)
Comment (givanovexpe) [reply README.md:38]: actually @rpoluri suggested adding cloudwatch in the variable name so to distinguish it from the more generic "log" type variables which are associated with logs on S3. This is for apiary-data-lake though. Here we don't store the logs on S3.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.2.0] - TBD
+
+### Added
+- Added `waggledance_logs_retention_days` variable that sets the `retention_days` of the Waggle Dance Cloudwatch group
```
Comment (massdosage) [inline CHANGELOG.md:9]:
```suggestion
- Added `waggledance_logs_retention_days` variable that sets the `retention_days` of the Waggle Dance Cloudwatch group.
```

**Comment by patduin** [comment]
closing, please reopen if still relevant

**Pull request closed without merging**